framework/src/testing/fixtures/genesis-asset.ts

```diff
@@ -693,75 +695,16 @@ export const blockAssetsJSON = [
 			supplySubstore: [
 				{
 					localID: '00000000',
-					totalSupply: '1000000000000000',
+					totalSupply: '1120000000000000',
```

Comment:
Is it possible to use `convertLSKToBeddows` here?

Reply:
same as above, i think lsk is not relevant / correct here, so I would prefer to keep it this way